src/TodoListItems.js

```diff
@@ -0,0 +1,17 @@
+import React from "react"
+
+export default function TodoListItems (props) {
+
+    //map function to create list items from todoList array
+    let listItem = props.todo.map(item => (
```

Comment:
I think we are suppose to put the map function in the `TodoList` file. You can think of it like this: `TodoListItem` only wants to know about 1 item.